### DISCUSSION TITLE
💖💫Thank You for the new Seq + OpenTelemetry updates 🚀 🌕

### DISCUSSION THREAD

**Discussion by PureKrome** [datalust/seq-tickets#1844] [General]
Just sending a pile of dev-ops hugs 🤗 to the entire Seq team for the Seq + OpenTelemetry love.

Great idea and looking forward to seeing more wins on the board, with these new updates.

Thanks all and keep it up!

![](https://media.giphy.com/media/DXHWqYa8xnOxwFV2dD/giphy.gif)

* not trolling. Sincere post.

**Comment by nblumhardt** [comment]
Thanks @PureKrome! Much appreciated! :-)
